### PR TITLE
Introductory docs for the dapp-runner

### LINKED ISSUE #36: intro docs

### PR DESCRIPTION
**Pull request by shadeofblue**
closes #36

### CODE REVIEW

**Review by nzmel — APPROVED** [review]

**Review by nzmel — APPROVED** [review]

**Review by nzmel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,265 @@
...
+
+`dapp-runner` is a utility that allows you to run decentralized applications on [Golem](https://www.golem.network/).
+It uses simple application descriptors expressed in `yaml`, similar to those used by
+tools like `docker-compose`.
+
+`dapp-runner` runs alongside the [Golem daemon](https://github.com/golemfactory/yagna)
+and uses [yapapi](https://github.com/golemfactory/yapapi), Golem's Python high-level API
+to communicate with it. As opposed to using plain `yapapi` though, deployment of
+applications on Golem using `dapp-runner` requires no code and no experience in Python.
+
```
Comment (stranger80) [inline README.md:11]: Would it be suitable to add a mention and reference to `dapp-manager`?

File: README.md [hunk 2]
```diff
@@ -1 +1,265 @@
...
+payment:
+  budget: 1.0  # GLM
+  driver: "erc20"
+  network: "rinkeby"
+```
+
+### Descriptors
+
+One or more application descriptors, as specified in  the
```
Comment (stranger80) [inline README.md:259]:
```suggestion
One or more application descriptors, as specified in the
```

**Review by stranger80 — CHANGES_REQUESTED** [review]
Good stuff. Minor suggestions added.

File: README.md (hunk 1 above)
Comment (shadeofblue) [reply README.md:11]: sure, the dependency goes the other way round but I think there'd be no harm

Comment (shadeofblue) [reply README.md:11]: added

**Review by stranger80 — APPROVED** [review]

**Pull request merged**